UPGRADING.md

```diff
@@ -15,6 +15,45 @@ end
 
 See [#957](https://github.com/intridea/grape/pull/957) for more information.
 
+#### Replace error_response with error! in rescue_from blocks
```

Comment:
As of now, no. Do you think the correct move moving forward is to remove error_response all together since its now undocumented? If so, I think its worth mentioning that it will be deprecated and adding a TODO to refactor error! to replace error_response.